submissions/EmmanuelUdoh/HTML/landingPage.html

```diff
@@ -0,0 +1,114 @@
+ <!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Kimaj:online shop for Electronics, Phones, Computers, Fashion ...</title>
+    <link rel="stylesheet" href="landing.css">
+
+</head>
+<body>
+    <div id="wrapper">
+        <div id="header">
+            <div id="subheader">
+                <div class="container">
+                    <p>We do business right: our customers, our priority</p>
+                    <a href="#">Download app</a><a href="#">Email support</a><a href="#">Help</a>
+                </div>
+            </div>
+
+        <!--==main header==-->
+        <div id="main-header">
+
+            <!--logo-->
+        <div id="logo">
+            <span id="ist">Kimaj</span>
+        </div>
+
+        <!--==search area==-->
+        <div id="search">
+            <form action="">
+                <input class="search-area" type="text" name="text" placeholder="search for products, brands and category">
+                <input class="search-btn" type="submit" name="submit" value="SEARCH">
+            </form>
+        </div>
+
+        <!--==user-menu==-->
+         <div id="user-menu">
+             <li><a href="#">Sell now</a></li>
+             <li><a href="#">Login/SignUp</a></li>
+             <li><a href="#">cart</a></li>
+         </div>
+        </div>
+
+        <!--== navigation bar==-->
+        <div id="navigation">
+            <nav>
+                <a href="#">All categories</a>
+                <a href="#">Electronics</a>
+                <a href="#">Phones and Tablets</a>
+                <a href="#">Computers and Accessories</a>
+                <a href="#">Fashion</a>
+                <a href="#">Kids and Toys</a>
+               
+            </nav>
+        </div>
+        <!--home banner-->
+        <div id="banner">
+            <ul class="banner">
+                <li><img src="images/b6.jpg"></li>
```

Comment:
Why is this a list item? It shows a huge dot right next to the banner in preview

Reply:
i have fixed this